cmd/snap-update-ns/utils.go

```diff
@@ -218,22 +196,27 @@ func (sec *Secure) MkPrefix(base string, perm os.FileMode, uid sys.UserID, gid s
 	}
 	iter.Next() // Advance iterator to '/'
 
-	const openFlags = syscall.O_NOFOLLOW | syscall.O_CLOEXEC | syscall.O_DIRECTORY
 	// Open the root directory and start there.
+	//
+	// NOTE: We don't have to check for possible trespassing on / here because
+	// we are going to check for it in sec.MkDir call below (which verifies
+	// that / is not violated)
```

Comment:
Reads a bit awkward. Maybe `.. which verifies that trespassing restrictions are not violated`?